openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -354,6 +354,8 @@ def _track_user_registration(user, profile, params, third_party_provider):
                 'is_year_of_birth_selected': bool(profile.year_of_birth),
                 'is_education_selected': bool(profile.level_of_education_display),
                 'is_goal_set': bool(profile.goals),
+                # Google Analytics only accept integers
+                'total_registration_time': int(params.get('totalRegistrationTime', '0').split('.')[0]),
```

Comment:
Send the floating value from MFE instead of a string a use `round()`.

Reply:
I tried sending that but it automatically makes it string. I only added `toString()` to MFE to be on the safe side if in case it send the float value the code here will break.